src/SDKs/Azure.Base/data-plane/Azure.Base.Tests/EventSourceTests.cs

```diff
@@ -108,5 +106,76 @@ public async Task SendingRequestProducesEvents()
             Assert.AreEqual(requestId, e.GetProperty<string>("requestId"));
             CollectionAssert.AreEqual(new byte[] { 6, 7, 8, 9, 0 }, e.GetProperty<byte[]>("content"));
         }
+
+        [Test]
+        public async Task NonSeekableResponsesAreLoggedInBlocks()
```

Comment:
This test seems to have a lot of responsibility.   Is there a way that it could be broken into several tests that each concentrate on validating one specific thing?

Reply:
Split into 2.